src/reference/environments/basic-3.7.yml

```diff
@@ -0,0 +1,8 @@
+name: cylc-8-basic-python-3.7
+# Cylc 8 without GUI installed from Conda Repos.
```

Comment:
Some envs say "installed from Conda Repos.", others don't which might be confusing.

```suggestion
# Cylc 8 without GUI
```